style.css

```diff
@@ -0,0 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Rock+Salt&display=swap');
+
+* {
+  margin: auto;
+  box-sizing: border-box;
+  font-family: 'Rock Salt', cursive;
+}
+
+a:visited,
+a:hover,
+a {
+  text-decoration: none;
+  color: #000000;
+}
+
+body {
+  display: flex;
+  flex-direction:column;
+  height: 100vh;
+  background-color: #008000;
+  font-family: 'Rock Salt', cursive;
+  text-align: center;
+  overflow-x: hidden;
+}
+
+header {
+  display: flex;
+  position:absolute;
```

Comment:
It's back!!!!

I see that you're using absolute here in conjunction with flex on the body again in order to position the clouds. 

You can get the same effect by setting the header to position: relative (so that your clouds are absolutely positioned relative to the header) and taking flex back off the body element. You could then give the main element some top margin to space it out correctly